maptemplate.c

```diff
@@ -1241,6 +1243,9 @@ static int processItemTag(layerObj *layer, char **line, shapeObj *shape)
       argValue = msLookupHashTable(tagArgs, "precision");
       if(argValue) precision = atoi(argValue);
 
+      argValue = msLookupHashTable(tagArgs, "padding");
+      if (argValue) padding = atoi(argValue);
```

Comment:
this is a bit nitpicking, but there's a potential security implication if the padding value is negative or too large, so some clamping wouldn't hurt here

Reply:
I've added the following guard. 

`padding > 0 && padding < 1000`

Not sure what to set as an arbitrary max value. I can add to the docs. Padding should always be greater than 0 though. Should an error be thrown or just ignore the value?

As far as I understand users can't add their own templates so any security implications would imply someone had access to the server already?

Reply:
> As far as I understand users can't add their own templates so any security implications would imply someone had access to the server already?

yes, security is a big word. Was mostly to avoid a crash if someone would put a crazy value. Any behavior other than crashing is OK to me.